src/app/@skolera/services/upload_files_with_presigned_Url.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { baseUrl } from '../../../environments/environment';
+
+
+@Injectable()
+export class UploadFilesWithPreSignedUrlService {
+    constructor(
+        private http: HttpClient,
+    ) { }
+
+    getPreSignedUrl(params:any) {
+        return this.http.post(`${baseUrl}/api/presigned_url`, params);
+    }
+    putFileToS3(body: File, presignedUrl: string){
+        return this.http.put(presignedUrl, body)
+    }
+    getFileFromS3(public_url: string){
```

Comment:
could you please use camelcase only, so `publicUrl` instead of `public_url`

Reply:
done